src/components/CastList.js

```diff
@@ -1,33 +1,31 @@
 import styled from 'styled-components';
-import PosterActor from './PosterActor';
 import defaultPoster from '../assets/images/profile.png';
+import PosterActor from './PosterActor';
 
 export default function CastList({ castList, listName }) {
   return (
     <>
-      <ListHeader>{listName}</ListHeader>
+      <ListHeader>{listName ? listName : ''}</ListHeader>
       {castList?.length >= 1 ? (
-        <StyledList role="list" listLength={castList?.length}>
+        <List role="list" listLength={castList?.length}>
```

Comment:
```suggestion
        <List role="list" listLength={castList.length}>
```
You are only rendering this, if the `castList.length >= 1` and then it should have a length. I don't think you need the optional chaining here.

Reply:
If i don't use optional chaining here, my App crashes. :-/